Project is unable to be build

I really like the idea behind the project especially for big source bases like various compilers or OSs. The problem I'm facing unfortunately is with regards to building it successfully.

I have tried both of recommended ways in README and both have some issues. In case of 2nd approach
> go install ./...
which I updated to `go get …`, but still end up in fixing a ton of dependency issues with go.mod. Sorry, I'm not familiar with Go to resolve issues from ground up so I've been fixing issues as they appear. Eventually end up with huge go.mod, which is still incomplete, because errors still appear.

As for the 1st approach of downloading it I'm also getting errors. Currently this is the error output I see:
```
$ go get -u github.com/nelhage/llama/cmd/...
go: downloading github.com/apache/thrift/lib/go/thrift v0.0.0-20210120171102-e27e82c46ba4
go: downloading golang.org/x/net v0.0.0-20210614182718-04defd469f4e
go: downloading golang.org/x/text v0.3.6
# github.com/fraugster/parquet-go/parquet
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:760:36: not enough arguments in call to iprot.ReadStructBegin
	have ()
	want (context.Context)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:765:55: not enough arguments in call to iprot.ReadFieldBegin
	have ()
	want (context.Context)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:779:25: not enough arguments in call to iprot.Skip
	have (thrift.TType)
	want (context.Context, thrift.TType)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:838:31: not enough arguments in call to iprot.ReadFieldEnd
	have ()
	want (context.Context)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:842:31: not enough arguments in call to iprot.ReadStructEnd
	have ()
	want (context.Context)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:849:31: not enough arguments in call to iprot.ReadBinary
	have ()
	want (context.Context)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:858:31: not enough arguments in call to iprot.ReadBinary
	have ()
	want (context.Context)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:867:28: not enough arguments in call to iprot.ReadI64
	have ()
	want (context.Context)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:876:28: not enough arguments in call to iprot.ReadI64
	have ()
	want (context.Context)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:885:31: not enough arguments in call to iprot.ReadBinary
	have ()
	want (context.Context)
../../../go/pkg/mod/github.com/nelhage/parquet-go@v0.3.1-0.20210416231405-1e924319d941/parquet/parquet.go:885:31: too many errors
```
Would you be so kind and recommend how I can build this project to test it out?
Thanks!

--- a/README.md
+++ b/README.md
@@ -38,7 +38,8 @@ pronounced on smaller workstations.
 - A Linux x86_64 machine. Llama only supports that platform for
   now. Cross-compilation should in theory be possible but is not
   implemented.
-- The [Go compiler](https://golang.org/dl/). Llama is tested on v1.16 but older versions may work.
+- The [Go compiler](https://golang.org/dl/). Llama is tested on v1.16
+  but older versions may work.
 - An [AWS account](https://aws.amazon.com/)
 
 ### Install llama
@@ -46,7 +47,7 @@ pronounced on smaller workstations.
 You'll need to install Llama from source. You can run
 
 ```
-go get -u github.com/nelhage/llama/cmd/...
+go install github.com/nelhage/llama/cmd/...@latest
 ```
 
 or clone this repository and run

--- a/go.mod
+++ b/go.mod
@@ -18,4 +18,5 @@ require (
 	golang.org/x/sys v0.0.0-20200930185726-fdedc70b468f
 )
 
-replace github.com/fraugster/parquet-go v0.3.0 => github.com/nelhage/parquet-go v0.3.1-0.20210416231405-1e924319d941
+// Uncomment this to work around https://github.com/fraugster/parquet-go/pull/28
+// replace github.com/fraugster/parquet-go v0.3.0 => github.com/nelhage/parquet-go v0.3.1-0.20210416231405-1e924319d941

